WordPress/Tests/CodeAnalysis/PerformantPHPFunctionsUnitTest.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// These calls are slow and type casting is more performant. Warning.
+intval( '10' );
+strval( 100 );
+floatval( '100' );
+boolval( 1 );
+doubleval( '100' );
+$var = 100;
```

Comment:
This line is redundant and adds no value to the tests.

Other than that, while it doesn't matter for the actual sniff, the test code does not reflect how these functions are commonly used (except for the last one). Casting a hard-coded value is redundant anyway, in that case, the hard-coded value should be declared with the correct type. Casting/use of these functions will generally only be done on variables or expressions.